api/graphql/reservation_units/reservation_unit_filtersets.py

```diff
@@ -101,6 +99,11 @@ class Meta:
         model = ReservationUnit
         fields = ["pk", "unit", "keyword_groups"]
 
+    # Hide archived reservation units
+    def filter_queryset(self, queryset):
```

Comment:
I'd add this functionality to `ReservationUnitsFilter`'s `resolve_queryset` method in `schema.py` 

Its clear it works from here also, but there in the schema file we've done already some filtering to the types

Reply:
Took me a while to figure out where to put this code so that it affects only GQL and not Django Admin. This was the first location that seemed to work 😄 

I'll move it! 👍